benefits/core/migrations/local_fixtures.json

```diff
@@ -69,7 +69,8 @@
     "fields": {
       "name": "senior",
       "label": "(MST) Senior Discount",
-      "group_id": "group123"
+      "group_id": "group123",
+      "enrollment_success_template": "enrollment/success--mst-non-agency.html"
```

Comment:
These names with `-non-agency` are a little confusing. I think we should keep the default here as `success--mst` (since it covers every other case) and then for agency cards override the name to `success--mst-courtesy-cards` etc.